onnxruntime/core/providers/cpu/rnn/uni_directional_lstm.h

```diff
@@ -57,6 +68,12 @@ class UniDirectionalLstm {
   void LoadPeepholeWeights(const gsl::span<const T>& peephole_weights);
   void LoadBias(const gsl::span<const T>& WbRb_values);
 
+  template <typename WeightT>
+  void ComputeImpl(const gsl::span<const T>& inputs, const gsl::span<const int>& sequence_lengths, int num_directions,
+                   const GemmWeights<WeightT>& input_weights, const GemmWeights<WeightT>& recurrent_weights, gsl::span<T>& outputs,
+                   gsl::span<T>& final_hidden_state, gsl::span<T>& final_cell_state, gsl::span<T>& all_cell_states,
+                   gsl::span<T>& output_iofc);
+
```

Comment:
shall we a comment for the bellowing two fields, which is not used by training_model=true.

  IAllocatorUniquePtr<T> output_iofc_ptr_;
  gsl::span<T> output_iofc_;